src/rpc/core_rpc_server.h

```diff
@@ -320,6 +322,9 @@ namespace cryptonote
 private:
     bool check_core_busy();
     bool check_core_ready();
+
+    template<typename Response>
+    void fill_sn_response_entry(Response &entry, const service_nodes::service_node_pubkey_info &sn_info);
```

Comment:
This template is only provided in the .cpp file, which is a little strange; any reason to not provide the template definition later in the header instead?  Or alternatively, make it a free function in the .cpp file that passes in `m_core`?

Reply:
It is a private method, so it is only meant to be used from inside the .cpp file, no reason to bloat the header file. I don't have a strong opinion about whether it should be a method or a free function though, can change it if you still prefer the latter.